Common/Interfaces/IOrderProperties.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+namespace QuantConnect.Interfaces
+{
+    /// <summary>
+    /// Contains additional properties and settings for an order
+    /// </summary>
+    public interface IOrderProperties
+    {
+        /// <summary>
+        /// Returns a new instance clone of this object
+        /// </summary>
+        IOrderProperties Clone();
```

Comment:
my mind rambling - it does seem slightly peculiar that the only method on this interface is a `Clone` method.

Reply:
Yes, seems strange, but for now the only `IOrderProperties` implementation is `InteractiveBrokersOrderProperties`, not sure when we'll add other implementations and if there may be common methods to be added to the interface.
The `Clone` method is the only one currently needed, as it is called internally by order methods.